hammr/commands/image/image.py

```diff
@@ -228,73 +228,34 @@ def do_deploy(self, args):
             if pimage == 2:
                 return 2
 
-            image_id = generics_utils.extract_id(pimage.imageUri)
-            if image_id is None or image_id == "":
-                printer.out("Image not found", printer.ERROR)
+            target_platform = ""
+            if pimage.targetFormat:
+                target_platform = pimage.targetFormat.name
+
+            if not is_targeted_cloud_compatible(pimage):
+                printer.out("Hammr only supports deployments for Amazon AWS and OpenStack.", printer.ERROR)
                 return 2
 
             if not self.is_pimage_ready_to_deploy(pimage):
                 printer.out("Published image with name '" + pimage.name + " cannot be deployed", printer.ERROR)
                 return 2
 
-            if not self.is_targeted_cloud_amazon(pimage):
-                printer.out("Hammr only supports deployments for Amazon AWS.", printer.ERROR)
+            file = generics_utils.get_file(doArgs.file)
+            if file is None:
```

Comment:
The method get_file already prints something when there is an error, so I did not put an additional message here